### DISCUSSION TITLE
HTML linting example

### DISCUSSION THREAD

**Discussion by grosch** [eslint/eslint#14926] [Q&A]
I'm trying to figure out how to flag any child of an `igx-input-group` element which is of type `input` and does not have the `igxInput` attribute.  Similarly, any `label` child which does not have the `igxLabel` attribute.

Seems like it should be simple, but every tutorial I've looked at is parsing typescript. Is there a good doc somewhere showing how to write a rule that is targeting HTML instead?

**Comment by nzakas** [comment]
ESLint doesn’t natively support HTML linting, so we don’t have docs for it.
